egs2/TEMPLATE/asr1/pyscripts/utils/mfa_format.py

```diff
@@ -276,9 +280,19 @@ def make_durations(args):
                 len(lab_paths) > 0
             ), f"The folder {wavs_dir} does not contain any transcription."
             for lab_path in lab_paths:
+                wav_path = lab_path.as_posix().replace(
+                    ".lab", ".wav"
+                )  # Assumes .wav files are in same dir as .lab files
+                if not os.path.exists(wav_path):
+                    logging.warning("There is no wav file for %s, skipping.", lab_path)
+                    continue
+                with sf.SoundFile(wav_path) as audio:
```

Comment:
Try with:
```python
data, orig_rate = sf.read(wav_path)
no_samples = int(data.shape[0] * (args.samplerate / orig_rate))  # Assuming 1-D
```
And remove importing Decimal at top

Reply:
Modify Lines 289~292 with the given comment

Reply:
Thanks for your suggestion. I used sf.SoundFile() instead of sf.read(), since in my tests it is faster but gives the same result.
If you have a specific reason to use sf.read() instead, I'll gladly replace it.

Reply:
Interesting.
Do you know what the difference is?

Reply:
Got it.
OK, then, for this use, we can just use SoundFIle()